src/modules/launcher/mod.rs

```diff
@@ -292,19 +300,26 @@ impl Module<gtk::Box> for LauncherModule {
                     );
                 } else {
                     send_async!(tx, ModuleUpdateEvent::ClosePopup);
+                    let mut minimize_window = false;
```

Comment:
I think it'd be slightly cleaner to calculate this in-place, like:

```suggestion
                    let mut minimize_window = false;
                    let minimize_window = matches!(event, ItemEvent::MinimizeItem(_));
```

(untested but I think that should work)